### ISSUE TITLE
BCELossBoud should add  a softmax layer!

### ISSUE THREAD

**Issue by Yanfeng-Zhou** [jianpengz/ConResNet#7]
In your code ConResNet/utils/loss.py

    class BCELossBoud(nn.Module):
        ...
       def forward(self, predict, target):
             predict = torch.softmax(predict, 1)

in forward section, I think we  should add predict = torch.softmax(predict, 1)
